.github/scripts/install_crds.sh

```diff
@@ -6,21 +6,19 @@
 # Requirements:
 # - Connection to a running k8s cluster (e.g. kind)
 
-install_appstudio_repo_crds () {
```

Comment:
Why? The function is hardcoded to the appstudio repo. What was wrong with the name?

Reply:
Restored.